modules/trino/src/main/java/org/testcontainers/containers/TrinoContainer.java

```diff
@@ -0,0 +1,109 @@
+package org.testcontainers.containers;
+
+import org.jetbrains.annotations.NotNull;
+import org.testcontainers.containers.wait.strategy.LogMessageWaitStrategy;
+import org.testcontainers.utility.DockerImageName;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.time.Duration;
+import java.util.HashSet;
+import java.util.Set;
+
+import static com.google.common.base.Strings.nullToEmpty;
+import static java.lang.String.format;
+import static java.time.temporal.ChronoUnit.SECONDS;
+
+/**
+ * @author findepi
+ */
+public class TrinoContainer extends JdbcDatabaseContainer<TrinoContainer> {
+    private static final String NAME = "trino";
+    private static final DockerImageName DEFAULT_IMAGE_NAME = DockerImageName.parse("trinodb/trino");
+    private static final String IMAGE = "trinodb/trino";
+    private static final String DEFAULT_TAG = "351";
+
+    public static final Integer TRINO_PORT = 8080;
```

Comment:
```suggestion
    private static final Integer TRINO_PORT = 8080;
```

Reply:
don't see why. I think the port constant may be useful and eg `org.testcontainers.containers.MySQLContainer#MYSQL_PORT` is public
i can still do the change, if you want me to

Reply:
`MySQLContainer` is quite old, nowadays we're trying to reduce the public API, including such constants :)